ignite/services/network/networkchain/prepare.go

```diff
@@ -104,13 +104,13 @@ func (c Chain) buildGenesis(
 
 	// apply genesis information to the genesis
 	if err := c.applyGenesisAccounts(ctx, gi.GenesisAccounts, addressPrefix); err != nil {
-		return errors.Wrap(err, "error applying genesis accounts to genesis")
+		return fmt.Errorf("error applying genesis accounts to genesis: %w", err)
```

Comment:
So I assume you have your word on https://discord.com/channels/893126937067802685/907904892780290058/1047801110863294484 :)